### PR TITLE
Modify readme to explain behaviour when encountering region-blocked videos and clarified date cutoffs

### LINKED ISSUE #31: I don't understand the distinction between "Archive", "Active", and "Watch"
I'm understanding mostly everything in the readme but I would appreciate some clarification on terminology. 

First on "Watch", which seems self-explanatory.

> Watch Script: Script that will download individual videos made by a creator that are not meant to be archived but deleted after being watched

I understand this as basically "throwaway" videos. So if I want to save a specific video(s), don't care about keeping it, then I use this script. For example tomorrow I will make my first (and last) pizza. I put 10 pizza tutorials in the .txt, they get downloaded, then I can delete them if I want (or not). Metadata that normally archivers want (such as thumbnail) are skipped. Is this correct?

>Archive Scripts: Scripts that will download everything that was created before December 31, 2019 (included).
>Active Scripts: Scripts that will download everything that was created after January 1, 2020 (included).

Sadly I don't understand the distinction between "Active" and "Archive". I've read it a few times and racked my brains hard before asking, so I would appreciate any clarification. Focusing specifically on Channels, (ignoring Playlists/Unique), the scripts appear to be identical between "Active" and "Archive" minus the cutoff date. 

It seems that this is more for organization purposes. I think what you are saying is that, not all a channel's content is worth keeping, for example live videos or deleted content. Essentially active/archive performs a filter. At the end of the year, I go through "active". I move the videos that are worth keeping into "archive" and delete the rest. To prevent deleted videos from being redownloaded using the archive script, I also append them to the respective archive.log. 

Then why does the "archive" *script* exist, rather than just an "archive" *folder*? For example if a channel that started in 2016 and is still active, 2016-2020 will go in archive, and 2021- will go in active. On the other hand with a channel ended in 2016, and the active/archive distinction is 2020/2021, placing the channel in either active/archive will download the same content. 

My question is, where do I put the channel I want saved? In "active", "archive", or both?

>These dates will need to be incremented every year January 1 and the content created by the active scripts will need to be moved to the archive (including the archive.log that will need to be merged).

Where does the date go, in the scripts?

### LINKED ISSUE #33: How will region-blocked videos be handled?
Let's say a video not available in my region is encountered. If I use a VPN, will it download the next time?

### PR DESCRIPTION
**Pull request by 93-infinity**
Closes #33 

Closes #31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,14 +43,16 @@ Downloading and archiving "live" content is complicated, especially because giga
 
 Considering that I chose to separate the content:
 
-Archive Scripts: Scripts that will download everything that was created before December 31, 2019 (included).
+Archive Scripts: Scripts that will download everything that was created before December 31 of the previous year, inclusive. For example, if the current date is any day between 2021-01-01 and 2021-12-31, everything created up to and including 2020-12-31 will be downloaded.
```
Comment (TheFrenchGhosty) [inline README.md:46]: inclusive > included

File: README.md [hunk 2]
```diff
@@ -43,14 +43,16 @@ Downloading and archiving "live" content is complicated, especially because giga
...
 
-Active Scripts: Scripts that will download everything that was created after January 1, 2020 (included).
+Active Scripts: Scripts that will download everything that was created after January 1 of the current year, inclusive. For example, if the current date is 2021-02-02, everything created between and during the dates 2021-01-01 and 2021-02-02 will be downloaded.
```
Comment (TheFrenchGhosty) [inline README.md:48]: inclusive > included

File: README.md [hunk 3]
```diff
@@ -67,6 +69,13 @@ Three different type of scripts are included depending of what you want to downl
 `Unique` : Made to download individual videos made by creators (naming scheme: `Title - Uploader - 20201231/Title - Uploader - 20191231 [VIDEOID].ext`).
 
 
+## Region-restricted videos?
+
+`--ignore-errors` and `--download-archive archive.log` will silently skip such videos.
+
+Using a VPN, switch between US and Germany. Or in general: if you follow a specific country's content, switch to that country (e.g. switch to France if you follow French content)
```
Comment (TheFrenchGhosty) [inline README.md:76]: Replace with:

Using a VPN, switch between US and Germany, and if you follow a specific country's content, also switch to that country (e.g. switch to France if you follow French content).

**Review by TheFrenchGhosty — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (93-infinity) [reply README.md:46]: pls clarify, I don't understand

**Pull request closed without merging**